cardano-cli/src/Cardano/CLI/Types/Errors/CmdError.hs

```diff
@@ -59,7 +58,6 @@ instance Error GovernanceActionsError where
 
 data EraBasedDelegationError
   = EraBasedDelegReadError !(FileError InputDecodeError)
-  | EraBasedCredentialError !ShelleyStakeAddressCmdError -- TODO: Refactor. We shouldn't be using legacy error types
```

Comment:
Deleting this constructor which isn't needed anymore after introducing `EraBasedDelegationStakeCredentialError` which also solves this problem https://github.com/input-output-hk/cardano-cli/pull/190/commits/f9727704bfa3cca3a46dd730b6fc2cd562631950#r1299054957